src/main/java/tsundere/task/Task.java

```diff
@@ -0,0 +1,61 @@
+package tsundere.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Initializes Task with its name.
+     * Completion status is false by default.
+     *
+     * @param description Name of Task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns completion status of Task.
+     *
+     * @return X if Task is done, blank otherwise.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    /**
+     * Sets Task completion status to true.
+     */
+    public void markAsDone() {
```

Comment:
Consider renaming the function to `markTaskAsDone()` so that the function becomes even more explicit.